.travis.yml

```diff
@@ -0,0 +1,29 @@
+language: scala
+
+scala:
+  - 2.11.8
+  - 2.12.0
+
+jdk:
+  - oraclejdk8
+
+sudo: required
+
+cache:
+  directories:
+    - $HOME/.ivy2/cache
+    - $HOME/.sbt/boot/
+
+before_cache:
+  - find $HOME/.ivy2 -name "ivydata-*.properties" -delete
+  - find $HOME/.sbt -name "*.lock" -delete
+
+script:
+  sbt ++$TRAVIS_SCALA_VERSION test
+
+after_success:
+  - if [[ ( "$TRAVIS_BRANCH" == "master" || "$TRAVIS_PULL_REQUEST" == "false" ) ]]; then
```

Comment:
Good point. In that case it would make sense to switch to similar versioning logic we use other projects, using git tag. I will make that change.